OpenVPN tunnel : update default cipher/digest values

kevincela opened the pull request:
- Update default cipher/digest values when adding new OpenVPN tunnel
- Fix delete button color in modal when deleting a tunnel

Card: https://trello.com/c/qicHdpv6/250-openvpn-tunnel-default-cipher-digest-values

andre8244 reviewed (changes requested):
- Need to set new default values at lines 290 and 291
- Change or remove comments at lines 217 and 233

kevincela commented:
> * Need to set new default values at lines 290 and 291
> * Change or remove comments at lines 217 and 233

I've removed the comments at lines 217 and 233. 

The default values at lines 290 and 291 are actually needed because these values are used when the item to edit does not have the `cipher` and `digest` values, in which case the `auto` option has to be selected. I've added a comment to attempt to clarify this.
